### DISCUSSION TITLE
Best practice to distinguish which versions are using environment-specific endpoints

### DISCUSSION THREAD

**Discussion by prandelicious** [fastlane/fastlane#19635] [Q&A] [answered]
Hi all,

When we build the app and use different sets of endpoints (e.g. development, testing, production), there's no way to easily distinguish which builds use which endpoints in TestFlight. What is the best practice for testing so users would easily distinguish which versions are using specific endpoints? Hope I'm clear enough.

Thanks in advance!

**Comment by lucgrabowski** [comment, marked as answer]
Hi, you could take a look at available plugins site https://docs.fastlane.tools/plugins/available-plugins/ and look for plugins allowing for adding icon overlays, e.g. `icon_versioning`, `badge` or `icon-banner`
